keps/sig-storage/0661-statefulset-volume-resize/kep.yaml

```diff
@@ -0,0 +1,57 @@
+title: StatefulSet volume resize
+kep-number: 0661
+authors:
+  - "@shalini-b"
+owning-sig: sig-apps
+participating-sigs:
+  - sig-storage
+status: provisional
+creation-date: 2021-07-13
+reviewers:
+  - "@kow3ns"
+  - "@gnufied"
+  - "@msau42"
+  - "@xing-yang"
+approvers:
+   - "@kow3ns"
+   - "@xing-yang"
+
+##### WARNING !!! ######
+# prr-approvers has been moved to its own location
+# You should create your own in keps/prod-readiness
+# Please make a copy of keps/prod-readiness/template/nnnn.yaml
+# to keps/prod-readiness/sig-xxxxx/00000.yaml (replace with kep number)
+#prr-approvers:
```

Comment:
Please add @deads2k as PRR approver here.

Reply:
Can you uncomment this line and add @wojtek-t as prr-approvers?

Reply:
Done